internal/services/disbursement_management_service_test.go

```diff
@@ -28,7 +28,7 @@ func Test_DisbursementManagementService_GetDisbursementsWithCount(t *testing.T)
 
```

Comment:
As stated in the other comment, we should remove the extra boolean flag for addUserMetadata, and pass in an `authManagerMock` when creating the service in this test. 

`service := NewDisbursementManagementService(models, models.DBConnectionPool, authManagerMock)`

Reply:
this test passing was a false positive - I fixed it now